test/SILGen/witness_tables.swift

```diff
@@ -1,9 +1,9 @@
 
-// RUN: %target-swift-emit-silgen -module-name witness_tables  -I %S/Inputs -enable-source-import %s -disable-objc-attr-requires-foundation-module -enable-objc-interop > %t.sil
+// RUN: %target-swift-emit-silgen -emit-sorted-sil -module-name witness_tables -I %S/Inputs -enable-source-import %s -disable-objc-attr-requires-foundation-module -enable-objc-interop > %t.sil
```

Comment:
Why the change to sorting here?

Reply:
The order differs between Linux/Darwin and Windows.  I don't think that there is a way to get both sides to work without the sorting.  Perhaps there is some other approach that I didn't consider.

Reply:
Hm. I'm pretty sure the order is supposed to be based on source, so that's a little suspicious.

Reply:
My fear about this is that the need for it could show a source of non-determinism.

Reply:
@jrose-apple, empirical observation indicates that this is not the case.  There is certainly a possibility of there being a bug in SILGen.